docs/site/managing_problems.md

```diff
@@ -26,7 +26,7 @@ click the _View on Site_ button.
 ![](http://i.imgur.com/ZgO5xcY.png)
 
 ## Editing Test Data
-Internally, the DMOJ uses a YAML-based format for describing problem data, which you may read about [here](/judge/problem_format.md).
+Internally, the DMOJ uses a YAML-based format for describing problem data, which you may read about [here](/problem_format/problem_format.md?id=problem-format).
```

Comment:
It would be nice to mention in the body of the PR why this `?id` is required. Also, it appears as though `#problem-format` has the same effect, and is more idiomatic.

Reply:
Thanks for the suggestions, will do!

Reply:
You should resolve this.